src/button/icon-helper.tsx

```diff
@@ -27,7 +27,7 @@ function getIconAlign(props: ButtonIconProps) {
 }
 
 function IconWrapper({ iconName, iconUrl, iconAlt, iconSvg, iconSize, ...props }: ButtonIconProps) {
-  if (!iconName && !iconUrl && !iconSvg) {
+  if (!iconName && !iconUrl && !iconSvg && !iconAlt) {
```

Comment:
I want to especially call out this change. It makes it so that icons that have iconAlt will be labeled regardless of other factors. This seems safe to me but I want folks who are more familiar to confirm

Reply:
If I understand this change correctly, it will cause all Icons to disappear if they don't have `iconAlt` set, right? That would be a breaking change.